utils/generate-command-code.py

```diff
@@ -166,13 +166,14 @@ def _flags_code():
                 s += "CMD_ARG_MULTIPLE_TOKEN|"
             return s[:-1] if s else "CMD_ARG_NONE"
 
-        s = "\"%s\",%s,%d,%s,%s,%s,%s" % (
+        s = "\"%s\",%s,%d,%s,%s,%s,%s,%s" % (
```

Comment:
maybe with a small extra effort we can avoid having this introduce changes to so many lines in commands.c? 
i.e. same trick as we did for `subargs` below.
(since that damn file is part of the repo, it can cause unnecessary conflicts to other PRs)